wikitemplate.md

```diff
@@ -145,6 +145,20 @@ _Each start should take less than 7 seconds_
   - [ ] Verify tips list is not lost after upgrade
   - [ ] Verify wallet panel transactions list is not lost after upgrade
 
+### Sync
+
+- [ ] Verify you are able to create a sync chain and add a mobile/computer to the chain
+- [ ] Verify you are able to join an existing sync chain using code words
+- [ ] Verify the device name is shown properly when sync chain is created
+- [ ] Verify you are able to add a new mobile device to the chain via QR code/code words
+- [ ] Verify new bookmarks added gets sync'd to all devices on the sync chain
```

Comment:
nit: "Verify `newly` created bookmarks `get` sync'd to all devices on the sync chain"